packages/lodestar/src/network/reqResp.ts

```diff
@@ -235,16 +237,16 @@ export class ReqResp extends (EventEmitter as IReqEventEmitterClass) implements
           }
           if (requestSingleChunk && responses.length === 0) {
             // allow empty response for beacon blocks by range/root
-            throw `No response returned for method ${method}`;
+            throw `No response returned for method ${method}. request=${requestId}`;
           }
           const finalResponse = requestSingleChunk ? responses[0] : responses;
-          this.logger.verbose(`receive ${method} response from ${peerInfo.id.toB58String()}`);
+          this.logger.verbose(`receive ${method} response from ${peerInfo.id.toB58String()}`,{requestId, encoding});
           return requestOnly ? null : finalResponse as T;
         }
       );
     } catch (e) {
       this.logger.error(
-        `failed to send ${method}(${encoding}) to peer ${peerInfo.id.toB58String()}.`, e
+        `failed to send request ${requestId} to peer ${peerInfo.id.toB58String()}`, e
```

Comment:
can we keep `method` and `encoding` in this log? Without it it's hard to debug issues like #972 when errors happen.

Reply:
you can search requestId in logs and you will find log where request came with it's method and encoding

Reply:
ah I see the log now